lib/src/lib/components/form-fields/amount-field/amount.pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { AmountOptions } from './amount-field.model';
+import {
+  LAB900_FORM_MODULE_SETTINGS,
+  Lab900FormModuleSettings,
+} from '../../../models/Lab900FormModuleSettings';
+import {
+  formatAmountWithoutRounding,
+  getAmountFormatter,
+} from './amount.helpers';
+
+@Pipe({
+  name: 'amount',
+})
+export class AmountPipe implements PipeTransform {
```

Comment:
this looks like it does the same as the `DecimalPipe` except using the settings from the forms module instead

Reply:
Thought about creating a decimal proxy pipe but I went with a new one just to keep full control.